src/app/ui/ChangelogPanel.jsx

```diff
@@ -6,17 +6,20 @@ import _        from 'lodash'
 import Panel    from 'bemuse/ui/Panel'
 import Markdown from 'bemuse/ui/Markdown'
 
-export const ChangelogPanel = React.createClass({
-  getInitialState () {
-    return { data: { status: 'loading' } }
-  },
+class ChangelogPanel extends React.Component {
+  constructor () {
+    super()
```

Comment:
Actually the constructor form should be:

```js
  constructor (props) {
    super(props)
```

Although [there is no difference if props is not accessed during construction](https://stackoverflow.com/a/34995257/559913), the documentation recommends [always passing `super(props)`](https://reactjs.org/docs/state-and-lifecycle.html#adding-local-state-to-a-class).

However, if all a constructor does is setting the state, it is better to remove a constructor entirely and use public class fields:

```js
  state = { data: { status: 'loading' } }
```